src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqBurdenEstimateRepository.kt

```diff
@@ -38,8 +42,42 @@ class JooqBurdenEstimateRepository(
         stochasticBurdenEstimateWriter: StochasticBurdenEstimateWriter? = null
 ) : JooqRepository(dsl), BurdenEstimateRepository
 {
-    private val centralBurdenEstimateWriter: BurdenEstimateWriter = centralBurdenEstimateWriter ?:
-            CentralBurdenEstimateWriter(dsl)
+    private fun getCountriesAsLookup(): Map<Short, String> = dsl.select(Tables.COUNTRY.ID, Tables.COUNTRY.NID)
+            .from(Tables.COUNTRY)
```

Comment:
The estimates come in with a country ISO code, which is `country.id` in the db, but they are stored in the `burden_estimate` table against a numeric index which is `country.nid`. So when reading the estimates out and comparing them to the expectations (which are stored against the ISO code) we need a lookup from numeric index to ISO code, which is what this is doing.